ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -682,5 +682,12 @@ final class Metrics(val registry: MetricRegistry) {
         val prune: Timer = registry.timer(Prefix :+ "prune")
       }
     }
+
+    object http_json_api {
+      private val Prefix: MetricName = daml.Prefix :+ "http_json_api"
+
+      val httpRequest: Timer = registry.timer(Prefix :+ "http_request")
+      val websocketRequest: Timer = registry.timer(Prefix :+ "websocket_request")
+    }
```

Comment:
The metrics have been collected here to make it easier to track and document them for the participant server.

For separate components (like the HTTP-JSON API) we should follow the same approach but use a different file.

For example, [this](https://github.com/digital-asset/daml/blob/2fa250f06acb3e1a037db94037dcce262556080b/runtime-components/non-repudiation/src/main/scala/com/daml/nonrepudiation/Metrics.scala) is the file collecting together the metrics for the non-repudiation middleware.

Reply:
ok adressed, hope it is fine.